--- netcdf/config.h.cmake.in	2019-11-21 00:59:32.000000000 +0100
+++ netcdf/config.h.cmake.in	2020-01-28 00:26:09.000000000 +0100
@@ -24,12 +24,6 @@
 
    */
 
-/* #cmakedefine HAVE_WINSOCK2_H
-
-#ifdef HAVE_WINSOCK2_H
-   #define _WINSOCKAPI_
-#endif
-*/
    #if _MSC_VER>=1900
      #define STDC99
    #endif
@@ -588,15 +582,11 @@
 /* Define to `unsigned long if <sys/types.h> does not define. */
 #cmakedefine uintptr_t unsigned long
 
-/* Define strcasecmp, snprintf on Win32 systems. */
+/* Define strcasecmp on Win32 systems. */
 #ifdef _WIN32
     #ifndef HAVE_STRCASECMP
 	 #define strcasecmp _stricmp
     #endif
-
-    #ifndef HAVE_SNPRINTF
-	 #define snprintf _snprintf
-    #endif
 #endif
 
 #cmakedefine WORDS_BIGENDIAN
